Tetris Clone C++/TetrisPiece.hpp

```diff
@@ -33,6 +33,7 @@ class TetrisPiece
     int x, y;
     // col, row are grid-based positions
     int col, row;
+    int * frameCounter;
```

Comment:
Should this be a pointer? I don't like having the Tetris piece look at variables that exist in the main loop.